zio-http-example/src/main/scala/example/CliExamples.scala

```diff
@@ -46,9 +46,8 @@ trait TestCliEndpoints {
     Endpoint
       .get(
         "users" / int("userId") ?? Doc.p("The unique identifier of the user") /
-          "posts" / int("postId") ?? Doc.p("The unique identifier of the post"),
+          "posts" / int("postId") ?? Doc.p("The unique identifier of the post") :? paramStr("The user's name"),
```

Comment:
```suggestion
          "posts" / int("postId") ?? Doc.p("The unique identifier of the post") :? paramStr("user-name"),
```

Reply:
Be sure to run this example since I don't think you're supplying the param on the other side so it would fail.